__mocks__/handlers/task-details.handler.ts

```diff
@@ -2,6 +2,43 @@ import { rest } from 'msw';
 const URL = process.env.NEXT_PUBLIC_BASE_URL;
 
 const taskDetailsHandler = [
+    rest.get(`${URL}/tasks/12345/details`, (_, res, ctx) => {
```

Comment:
1. The first argument _ in the callback function seems unused.
2.   In the taskData object, you have values like "null" for links and dependsOn. It's generally better to represent missing data with an empty array ([]) or null.
3.  can we wrap into try...catch

Reply:
1. The arg is unused and so by convention denoted it using _.
2. accepted

Reply:
@skv93-coder what about point 3

Reply:
3. accepted